c-sharp.md

```diff
@@ -6,7 +6,15 @@ template = "page_lang"
 [extra]
 author = "Fermyon Staff"
 last_modified = "2022-03-10T21:50:50Z"
+
 ---
+
+- [C# and .NET in WebAssembly](#c-and-net-in-webassembly)
+	- [Available Implementations](#available-implementations)
```

Comment:
This indent looks different from the previous page - is that just GH UI silliness?

Reply:
Yes, that is normal (using the ToC extension in Visual Studio as per usual)
Tested H1, H2 and H3 and all seem to be working as intended.

Reply:
Probably some sort of tabs vs spaces shenanigans.  Thanks for checking!